modules/ROOT/pages/fd-share-to-studio.adoc

```diff
@@ -6,28 +6,34 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-When you are creating a Mule application, there might come a time when you would like help from someone in your Anypoint-Platform organization, such as a developer, who works in Studio. You can download your project as a `.jar` file and then place that file in a location from which the other person can import it into Studio.
+Perhaps when you are creating a Mule application you need help from someone in your Anypoint-Platform organization,
+such as a developer, who works in Studio. You can download your project as a `.jar` file and then place that file in a location from which the other person can
+import it into Studio.
 
-As the other person works on the project, the changes are applied to your project online in Flow Designer. When the other person is finished, you can open your project in Flow Designer and pick up where that person left off.
+You'll need to use *Export project sources* to share just your project source files (including all the connector credentials)
+without all of the other artifacts that are required by the Mule runtime.
 
-== Procedure
+As the other person works on the project, they can take advantage of the advanced features of Studio.
+When they are finished, they can choose to deploy the project from Studio.
```

Comment:
Consider adding something about the advanced features of Studio into a previous sentence.
Not sure we want the second sentence here: does this mean that the user-audience does NOT deploy, or cannot deploy?

Reply:
These are features that are not in FD. It's a lot to mention here. Skipping for now.